server/lib/device/device.create.js

```diff
@@ -119,6 +119,21 @@ async function create(device) {
       // if not, we create it
       feature.device_id = deviceToReturn.id;
       const featureCreated = await db.DeviceFeature.create(feature, { transaction });
+
+      // if feature state are attached, save feature state
+      if (feature.feature_state) {
+        feature.feature_state.forEach(async (state) => {
```

Comment:
I'm not sure, but async/await in a foreach loop doesn't work...

Reply:
in this case both works, i remove await

Reply:
If await is not useful, neither is async ;)